engines/director/score.cpp

```diff
@@ -564,6 +564,7 @@ void Score::renderFrame(uint16 frameId, RenderMode mode) {
 		setLastPalette(frameId);
 		renderSprites(frameId, mode);
 		_window->render();
+		resetNonPuppetSprites(frameId);
```

Comment:
Please explain why are you doing this? I thought, that this reset is happening in Channel::setClean()

```C++
		if (_sprite->_puppet || (!nextSprite->isQDShape() && partial)) {
			// Updating scripts, etc. does not require a full re-render
			_sprite->_scriptId = nextSprite->_scriptId;
		} else {
			previousCastId = _sprite->_castId;
			replaceSprite(nextSprite);
		}
```

e.g. we are not touching puppetized sprites